src/utils/ctl/SConscript

```diff
@@ -12,12 +12,13 @@ SRC = ['cart_ctl.c']
 
 def scons():
     """scons function"""
-    Import('env', 'prereqs', 'cart_lib', 'gurt_lib')
+    Import('env', 'prereqs', 'cart_lib', 'gurt_lib', 'cart_test_lib')
 
     tenv = env.Clone()
     tenv.AppendUnique(CPPPATH=['#/src/cart', '#/src/tests/ftest/cart'])
-    tenv.AppendUnique(LIBS=['cart', 'gurt', 'pthread'])
+    tenv.AppendUnique(LIBS=['cart', 'gurt', 'pthread', 'cart_test'])
     prereqs.require(tenv, 'mercury')
+    tenv.AppendUnique(LIBPATH=[Dir('../../tests/ftest/cart')])
```

Comment:
I think we should find a better place for cart_test sources.  If the code is shared by utilities that we actually ship, it should probably be a library like libcart_util.so and live with cart sources.  Either that or we should extract product code from test code.

Reply:
Rework of all the changes.  test_common.c(.h) are renamed to cart_utils.c(.h). The are moved to different directories and the name of the library has been changed.